src/Transport/NServiceBus.AzureServiceBus.csproj

```diff
@@ -12,7 +12,6 @@
   <PropertyGroup>
     <PackageId>NServiceBus.Azure.Transports.WindowsAzureServiceBus</PackageId>
     <Description>NServiceBus Transport for the Windows Azure Servicebus</Description>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
```

Comment:
I'd comment this out rather than remove.
For quick local verification, it's handy to be able to produce a package locally.